src/hooks/useLoadInitialData.ts

```diff
@@ -22,6 +23,8 @@ export const useLoadInitialData = () => {
     const drives = usePathContent();
     const prevDrivesState = useRef([...drives]);
 
+    useLoadDefaultDrive();
```

Comment:
I've integrated this feature into the render process instead of an [alternative location](https://github.com/powerhouse-inc/document-model-electron/blob/ebc020405e0b289aa4cfdf4935b2afcd53494a7f/src/app/document-drive.ts#L32). This is because [this particular call](https://github.com/powerhouse-inc/document-drive/blob/main/src/server/index.ts#L423) can occasionally take longer in respond, taking upwards of 5 seconds to complete. Implementing it in this manner ensures that the UI loading is not hindered while awaiting the response from this call.

Reply:
This is also required because this way the drive is also added in the browser when the electron process is not available.
The [alternative location](https://github.com/powerhouse-inc/document-model-electron/blob/ebc020405e0b289aa4cfdf4935b2afcd53494a7f/src/app/document-drive.ts#L32) only applies to the Electron document drive.